terragrunt/modules/sync-team/lambdas/start-sync-team/index.js

```diff
@@ -0,0 +1,8 @@
+const AWS = require('aws-sdk');
```

Comment:
Why are we using JavaScript for this one and Python for the other lambda?

Reply:
I remember spending hours to try and get the Python version to work without success, and switching to js fixed the problem.